webmails/start.py

```diff
@@ -51,7 +51,9 @@
         print(f"Can't read SECRET_KEY from file: {exc}", file=sys.stderr)
         exit(2)
 
-context['SECRET_KEY'] = hmac.new(bytearray(secret_key, 'utf-8'), bytearray('ROUNDCUBE_KEY', 'utf-8'), 'sha256').hexdigest()
+context['ROUNDCUBE_KEY'] = hmac.new(bytearray(secret_key, 'utf-8'), bytearray('ROUNDCUBE_KEY', 'utf-8'), 'sha256').hexdigest()
+context['SNUFFLEUPAGUS_KEY'] = hmac.new(bytearray(secret_key, 'utf-8'), bytearray('SNUFFLEUPAGUS_KEY', 'utf-8'), 'sha256').hexdigest()
```

Comment:
Where does `SNUFFLEUPAGUS_KEY` come from?

Reply:
This is just a salt for the KDF... so that we ensure we derive a separate key; could be anything.

Reply:
It's used here: https://snuffleupagus.readthedocs.io/config.html#global

Reply:
Yes, sorry - I've misread the line. It's fine.